menpo/visualize/image.py

```diff
@@ -4,6 +4,27 @@
 from menpo.feature import ndfeature
 
 
+@ndfeature
+def sum_channels(pixels, channels=None):
+    r"""
+    Sum the channels of an image.
+
+    Parameters
+    ----------
+    pixels : `ndarray`
+        The input image's pixels.
+    channels : `int`
+        Defines the channels to be summed.
+    """
+    # if channels is None, then all channels are used
+    if channels is None:
+        channels = range(pixels.shape[0])
+    # compute sum image
+    sum_image = np.sum(pixels[channels], axis=0)
+    # return as c-contiguous
+    return np.ascontiguousarray(sum_image[None, ...])  # add a channel axis
```

Comment:
``s.reshape((1,) + s.shape)`` does the same thing without requiring the copy. So you can remove the ``np.ascontiguousarray`` as well.